### PR TITLE
Feign bootstrapping agreement, 2024-05-27

### PR DESCRIPTION
**Pull request by velo**

### CODE REVIEW

File: PROJECTS.yaml [hunk 1]
```diff
@@ -103,3 +103,13 @@ sdkman:
     SDKMAN! is a robust tool designed for managing parallel versions of multiple Software Development
     Kits (SDKs) on Unix-based systems. An intuitive command-line interface simplifies the installation,
     updating, and switching of executables for various SDKs, including Java, Kotlin, Scala, and Groovy.
+
+feign:
+  name: Feign
+  home: https://github.com/OpenFeign/feign
+  repo: https://github.com/OpenFeign/feign
+  logo:
```
Comment (kenfinnigan) [inline PROJECTS.yaml:111]: Is there a logo you could add?

Having a logo means the project page can generate using it

Comment (velo) [reply PROJECTS.yaml:111]: Sadly, there is no logo.

I would be open for suggestions

Comment (kenfinnigan) [reply PROJECTS.yaml:111]: No problem

**Comment by kenfinnigan** [comment]
Is it only the feign project wanting to join the foundation? Will the other projects in the OpenFeign GitHub org remain?

**Comment by velo** [comment]
> Is it only the feign project wanting to join the foundation? Will the other projects in the OpenFeign GitHub org remain?

feign-form is maintained mostly by someone else and got inactive, maybe I should fold that into feign and close the project.

querydsl is a tricky subject.  Original project was dead/dying.  Tried to help to keep it alive and was told to pack sand.  So I forked and keep it as active as I can, but by no means I have any rights to give it away

everything else are more or less experiments with varying levels of success.

**Comment by aalmiray** [comment]
On a tangent note, I'd love to see feign-form revitalized.

**Comment by velo** [comment]
> On a tangent note, I'd love to see feign-form revitalized.

Could be the first initiative under commonhaus =)

**Review by cealsair — APPROVED** [review]

**Review by aalmiray — APPROVED** [review]

**Review by gavinking — APPROVED** [review]

**Pull request merged**